docs/sources/helm-charts/mimir-distributed/configure/configure-gem-federation-frontend.md

```diff
@@ -0,0 +1,125 @@
+
+   Note that these resource settings are examples that are sufficient for small deployments. Adjust the values based on your specific requirements and load:
+
+   ```yaml
+   # Enable enterprise features
+   enterprise:
+     enabled: true
+
+   # Enable and configure federation-frontend
+   federation_frontend:
+     enabled: true
+     # Since this is a standalone deployment, we configure the chart to not render any of the other GEM components.
+     disableOtherComponents: true
+     replicas: 2
+     resources:
+       requests:
+         cpu: 100m
+         memory: 128Mi
+       limits:
+         cpu: 1
+         memory: 256Mi
+     extraEnvFrom:
+       - secretRef:
+           name: gem-tokens
+
+   # Configure the remote GEM clusters to query.
+   mimir:
+     structuredConfig:
+       auth:
+         type: trust
+       # The federation-frontend doesn't handle authentication and authorization. Disabling multitenancy means the federation-frontend will not require the X-Scope-OrgID header.
+       # With disabled multitenant the federation-frontend will use the authn/z material from the proxy_targets configuration.
+       # For a complete list of configuration options, refer to the configuration reference at https://grafana.com/docs/enterprise-metrics/<GEM_VERSION>/config/reference/#federation.
+       multitenancy_enabled: false
+       federation:
+         proxy_targets:
+           - name: "cluster-1"
+             url: "http://gem-query-frontend.monitoring.svc.cluster.local:8080/prometheus"
+             basic_auth:
+               username: tenant-1
+               password: "${CLUSTER_1_GEM_TOKEN}"
+
+           - name: "cluster-2"
+             url: "https://gem.monitoring.acme.local/prometheus"
+             basic_auth:
+               username: tenant-2
+               password: "${CLUSTER_2_GEM_TOKEN}"
+
+   # Disable MinIO
+   minio:
+     enabled: false
+
+   # The federation-frontend doesn't need the rollout-operator for rollouts, so it can be disabled.
+   rollout_operator:
+     enabled: false
+   ```
+
+4. Deploy the federation-frontend using `helm`:
+
+   ```bash
+   helm install federation-frontend grafana/mimir-distributed -f federation-frontend.yaml -n federation-frontend-demo
+   ```
+
+   This will deploy only the federation-frontend component. The federation-frontend will be configured to proxy queries to the GEM clusters specified in the `proxy_targets` configuration.
+
+5. Verify that the federation-frontend is running. The simplest way to do this is to issue a label names query against the federation-frontend service.
+
+   This example tries to reach the Kubernetes service from the cluster and request the label names from the past 1 year.
+
+   ```bash
+   curl -XPOST 'https://mimir-federation-frontend:8080/prometheus/api/v1/labels' \
+     -d "start=$(date -u +%Y-%m-%dT%H:%M:%S.0Z -d '1 year ago' 2>/dev/null || date -u -v -1y +%Y-%m-%dT%H:%M:%S.0Z)" \
+     -d "end=$(date -u +%Y-%m-%dT%H:%M:%S.0Z)"
+   ```
```

Comment:
One year seems like a lot of data to query - how about the last 12 hours by default?